composer.json

```diff
@@ -48,9 +48,12 @@
         "psalm/plugin-phpunit": "0.16.1",
         "squizlabs/php_codesniffer": "3.6.1",
         "symfony/cache": "^5.2|^6.0",
-        "symfony/console": "^2.0.5|^3.0|^4.0|^5.0|^6.0",
+        "symfony/console": "^2.7|^3.0|^4.0|^5.0|^6.0",
         "vimeo/psalm": "4.13.0"
     },
+    "conflict": {
+        "symfony/console": "<2.7"
```

Comment:
Why is a conflict needed given that there's already a dependency on `^2.7`? Note, the dependency is development-only but the conflict will also prevent installing the production dependency.

Reply:
Correct. The console dependency is optional, but _if_ it's installed, I want it to be at least at version 2.7. This is by the way not a tough constaint. 2.7 was released in 2015 and is EOL for quite a while already.

The reason is that the classes I'm using to draw the table were introduced in version 2.7.

Reply:
I understand that the likelihood of the conflict is relatively low but given that it's a development dependency (in the sense that a developer will use it while developing an application, not necessarily the DBAL itself), I'd rather leave the dependency constraint resolution to their discretion (especially given that it affects a single command) than mess with their production dependency constraints.

I did not like the need to jump through the hoops needed to address the conflict between `symfony/console` and `psr/log >=3` and I do not want to do that to others.

Reply:
All right.